dataflow/flex-templates/pipeline_with_dependencies/flex_template_cloudbuild.yaml

```diff
@@ -0,0 +1,27 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Workflow that builds a template defined by Dockerfile.flex_template using a custom base image.
+
+steps:
+- name: 'gcr.io/cloud-builders/docker'
```

Comment:
shall we use `gcr` here or `ar`?

Reply:
this is still accurate as per https://cloud.google.com/build/docs/building/build-containers. this is mostly a URN